db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ OPTIONAL ] I highly recommend to add some of seeds to your app so anyone can run the seed to see the random messages so you can create them from there it will be easier for anyone to just run `rails db:seed` and after running your app he can see them 🤩✨💯🔥

Reply:
Thanks for the constructive feedback!